lua/neo-tree/defaults.lua

```diff
@@ -367,6 +367,8 @@ local config = {
       ["p"] = "paste_from_clipboard",
       ["c"] = "copy", -- takes text input for destination, also accepts the config.show_path and config.insert_as options
       ["m"] = "move", -- takes text input for destination, also accepts the config.show_path and config.insert_as options
+      ["Y"] = "copy_path",
+      ["gy"] = "copy_abspath",
```

Comment:
The only problem here is that the commands are defined at the global level so that it applies to all sources, but it is not really a command that will work universally. It should be mapped within the `filesystem` and `buffers` commands instead.

It also makes sense to use it in the `git_status` source.

Reply:
Forgive my ignorance, and would you explain it in more detail,thx alot